### PR TITLE
[15.0][MIG] onchange_helper

### PR DESCRIPTION
**Pull request by koenloodts**

### CODE REVIEW

**Review by BT-aestebanez — APPROVED** [review]

**Review by ghost — APPROVED** [review]
lgtm

**Review by BT-mgomez — APPROVED** [review]

**Comment by ghost** [comment]
@oerp-odoo Can you review this, please?

File: onchange_helper/models/base.py [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+            if field not in all_values:
+                all_values[field] = record_values.get(field, False)
+
+        new_values = {}
+        for field in onchange_fields:
+            onchange_values = self.onchange(all_values, field, onchange_specs)
+            new_values.update(self._get_new_values(values, onchange_values))
+            all_values.update(new_values)
+
+        return {
```
Comment (oerp-odoo) [inline onchange_helper/models/base.py:62]: I would rewrite this with normal `for` loop. As its cluttered with many iterations and its hard to read.

Comment (BT-aestebanez) [reply onchange_helper/models/base.py:62]: @koenloodts Could you apply this change so this can be merged?

Comment (BT-aestebanez) [reply onchange_helper/models/base.py:62]: @oerp-odoo Maybe we can merge this and create another PR with the refactoring of the for loop, what do you think? I could create it and assign you as reviewer

Comment (oerp-odoo) [reply onchange_helper/models/base.py:62]: Well, fine to me, but I can't merge:)

Comment (BT-aestebanez) [reply onchange_helper/models/base.py:62]: @sebastienbeau I saw you merged https://github.com/OCA/server-tools/pull/1936 , could you share your thoughts about this comment and merge the PR?

**Comment by BT-aestebanez** [comment]
@dreispt Can you merge this please?

**Pull request merged**